deploy/charts/rook-ceph/values.yaml

```diff
@@ -7,7 +7,7 @@ image:
   repository: rook/ceph
   # -- Image tag
   # @default -- `master`
-  tag: master
+  tag: v1.13.6
```

Comment:
Due to #13843 let's set the tag with the release version PR. Even though the release script was updating it for the official helm release, users are still having a problem with this tag when using this as the base template for their values override file.